mirdata/datasets/tiny_4mula.py

```diff
@@ -0,0 +1,319 @@
+            jams.JAMS: the track's data in jams format
+
+        """
+        return jams_utils.jams_converter(
+            spectrogram_path=self.melspectrogram_path,
+            metadata={
+                "art_id": self.art_id,
+                "art_name": self.art_name,
+                "art_rank": self.art_rank,
+                "main_genre": self.main_genre,
+                "music_id": self.music_id,
+                "music_lang": self.music_lang,
+                "music_lyrics": self.music_lyrics,
+                "music_name": self.music_name,
+                "musicnn_tags": self.musicnn_tags,
+                "related_art": self.related_art,
+                "related_genre": self.related_genre,
+                "related_music": self.related_music,
+                "duration": 30,  # approximation
+            },
+        )
+
+
+def load_melspectrogram(fhandle: str) -> ndarray:
+    """Load a tiny_4mula dataset melspectrogram.
+
+    Args:
+        fhandle (str): path pointing to the dataset file
+
+    Returns:
+        numpy.ndarray: melspectrogram
+
+    """
+    print(str(fhandle))
+    if not os.path.isfile(str(fhandle)):
+        raise FileNotFoundError("Dataset not found. Did you run .download()?")
+    df = read_parquet(fhandle, columns=["melspectrogram"])
+    return df["melspectrogram"].values
+
+
+def load_by_columns(fhandle: str, columns: list = DEFAULT_COLUMNS):
+    """Load a tiny_4mula dataset filtering by columns.
+
+    Args:
+        fhandle (str): path pointing to the dataset file
+        columns (list): attributes to filter in dataset read
+
+    Returns:
+        pandas.DataFrame: tiny_4mula dataset filtered by columns
+
+    """
+    if not os.path.isfile(str(fhandle)):
+        raise FileNotFoundError("Dataset not found. Did you run .download()?")
+    return read_parquet(fhandle, columns=columns)
+
+
+@core.docstring_inherit(core.Dataset)
+class Dataset(core.Dataset):
+    """
+    The tiny version of 4MuLA dataset
+    """
+
+    def __init__(self, data_home=None):
+        super().__init__(
+            data_home,
+            name="tiny_4mula",
+            track_class=Track,
+            bibtex=BIBTEX,
+            remotes=REMOTES,
+            download_info=DOWNLOAD_INFO,
+            license_info=LICENSE_INFO,
+        )
+
+    @core.cached_property
+    def metadata(self):
+        metadata_path = os.path.join(self.data_home, "4mula_tiny.parquet")
+        if not os.path.exists(metadata_path):
+            raise FileNotFoundError("Metadata not found. Did you run .download()?")
+        columns = [
+            "music_id",
+            "music_name",
+            "music_lang",
+            "music_lyrics",
+            "art_id",
+            "art_name",
+            "art_rank",
+            "main_genre",
+            "related_genre",
+            "related_art",
+            "related_music",
+            "musicnn_tags",
+        ]
+        return read_parquet(metadata_path, columns=columns)
```

Comment:
The `_metadata` should be a dictionary withe the `track_ids` as keys. Please see [this as example](https://github.com/mir-dataset-loaders/mirdata/blob/master/mirdata/datasets/cante100.py#L356-L406), and remember that we use `csv` to load tables. Let me know if you need further information!